tests/_simulatedbindings.py

```diff
@@ -49,15 +86,26 @@ def int_to_str(i):
     return rv
 
 
-def wrapListWithFormat(v, fmt, sep=' '):
+def _wrapListWithFormat(v: Sequence_T, fmt: str, sep: str = ' ') -> str:
+    '''
+    Specially format a list as a string
+
+    Args:
+        v: List
```

Comment:
Based on the typing, it looks like this could be anything w/ the `sequence` interface?

Reply:
updated